<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSimulator</class>
 <widget class="QMainWindow" name="NetworkSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetworkSimulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>195</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>start simulation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="protocol">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>79</width>
      <height>36</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>OSPF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RIP</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="cycleDuration">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="lambda">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>42</width>
      <height>37</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tickDurationShow">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tickCountShow">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QSlider" name="tickCount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>tick duration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>tick count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>58</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>lamdbda</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
